api/walletApi-V0.yml

```diff
@@ -180,19 +180,53 @@ paths:
       produces:
         - application/json
       responses:
-        "204":
+        204:
           description: Account deletion request has been taken into account, the user will be prompt to confirm the deletion.
           schema:
             $ref: "#/definitions/Account"
-        "400":
+        400:
           description: Bad request.
           schema:
             $ref: "#/definitions/Error"
-        "404":
+        404:
           description: Not found.
           schema:
             $ref: "#/definitions/Error"
-        "500":
+        500:
+          description: Internal Server Error - The server has encountered a situation it does not know how to handle.
+          schema:
+            $ref: "#/definitions/Error"
+    put:
+      operationId: ModifyAccountNickname
```

Comment:
I think this endpoint should be more generic. do not restrict to nickname. We will probably need to edit some others Account stuffs in the futur
The discription could be something like update account ...